ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -209,6 +222,18 @@ public void shouldFailToDescribeTopicsWhenRetriesExpire() {
     verify(adminClient);
   }
 
+  @Test(expected = TopicAuthorizationException.class)
+  public void shouldFailToDescribeTopicAuthorizationException() {
```

Comment:
nit:
```suggestion
  public void shouldPropagateTopicAuthorizationExceptionOnDescribe() {
```
or something similar. (The existing name makes it sound like the test tries to describe an exception.)